pages/Configuring/Variables.md

```diff
@@ -103,18 +102,20 @@ Using `blur_new_optimizations` with an animated wallpaper may actually increase
 
 {{< /hint >}}
 
-## Animations
+# Animations
 
-| name | description | type | default |
-|---|---|---|---|---|
-| enabled | enable animations | bool | true |
+| name    | description       | type | default |
+| ------- | ----------------- | ---- | ------- |
```

Comment:
again, asked you to revert this

Reply:
and I asked you to fix your stupid compositor, we don't always get what we want